naptime-graphql/src/test/scala/org/coursera/naptime/ari/graphql/helpers/ArgumentBuilder.scala

```diff
@@ -0,0 +1,43 @@
+package org.coursera.naptime.ari.graphql.helpers
+
+import org.coursera.naptime.PaginationConfiguration
+import sangria.schema.Args
+import sangria.schema.Argument
+import sangria.schema.IntType
+import sangria.schema.OptionInputType
+import sangria.schema.StringType
+
+import scala.collection.concurrent.TrieMap
+
+object ArgumentBuilder {
+  def buildArgs(argumentDefinitions: List[Argument[_]], argumentInputs: Map[String, Any]): Args = {
+    val argsWithDefault = argumentDefinitions.filter(_.defaultValue.isDefined).map(_.name).toSet
+    val optionalArgs = argumentDefinitions.filter(_.inputValueType.isOptional).map(_.name).toSet
+    val undefinedArgs = argumentDefinitions.map(_.name)
+      .filterNot(argsWithDefault.contains)
+      .filterNot(optionalArgs.contains)
+      .filterNot(argumentInputs.contains)
+      .toSet
+    val defaultInfo = argumentDefinitions
+      .filter(x => argsWithDefault.contains(x.name)).map(x => x.name -> x.defaultValue.get._1)
+    val builder = TrieMap.newBuilder[String, Any]
```

Comment:
i think instead of using a builder, you can do
```scala
TrieMap() ++= defaultInfo
```
if that doesn't work, let me know and we'll play with the types and stuff to get something like that working. take a look at https://www.scala-lang.org/api/current/scala/collection/concurrent/TrieMap.html too to see if there's other useful stuff we could do